app/navbar.py

```diff
@@ -7,24 +7,58 @@
 # Imports
 
 import dash_bootstrap_components as dbc
-
+import dash_html_components as html
+from pathlib import Path
+import base64
 
 # ------------------------------------------------------------------------------
 # Content
 
+
+# Encoding logo image file
+pyro_logo = Path(__file__).parent.joinpath('data', 'pyro_logo.png')
+encoded_image = base64.b64encode(open(pyro_logo, 'rb').read())
+
+
 def Navbar():
 
-    navbar = dbc.NavbarSimple(
+    # Dropdown menu
+    dropdown = dbc.DropdownMenu(
         children=[
-            dbc.DropdownMenu(nav=True,
-                             in_navbar=True,
-                             label='Tableaux de bord',
-                             children=[
-                                 dbc.DropdownMenuItem('Alertes et Infrastructures', href='alerts'),
-                                 dbc.DropdownMenuItem('Niveaux de Risque', href='risks')],
-                             )],
-        brand='À propos',
-        brand_href='/home',
-        sticky='top')
+            dbc.DropdownMenuItem('Alertes et Infrastructures', href='alerts'),
+            dbc.DropdownMenuItem('Niveaux de Risque', href='risks'),
+        ],
+        label="Tableaux de bord",
+        className="ml-auto flex-nowrap mt-3 mt-md-0 btn-group dropleft",
```

Comment:
I am not sure to understand what the `className` defined here (and I see a similar one in the instantiation of the `user_item` object) is designed for and where it comes from. Could you please describe briefly its use or point us to any related resource? Many thanks!

Reply:
Hi @pechouc here some explanations about those bootstrap css classes : 

- ml-auto : margin left auto in regards of other columns on the same row
- flex-nowrap force inline alignment as explained [here](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-wrap)
- mt et mt-md for margin top 
- btn-group dropleft to ensure that the dropdown will occur one the left (to match responsive needs) as detailed [here](https://getbootstrap.com/docs/4.0/components/dropdowns/)

Hope that helps